src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__image" src="./images/imac.jpeg" alt="product-image">
+
+      <div class="card__product-info">
+        <h2 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="stars-Wrapper">
+          <div class="stars-Wrapper__row stars stars--4">
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+          </div>
+          <span class="stars-Wrapper__card-reviews">
+            Reviews: 5
+          </span>
+
+        </div>
```

Comment:
```suggestion
        <div class="card__rating">
          <div class="stars stars--4">
            <div class="stars__star star"></div>
            <div class="stars__star star"></div>
            <div class="stars__star star"></div>
            <div class="stars__star star"></div>
            <div class="stars__star star"></div>
          </div>
          <span class="card__rating-reviews">
            Reviews: 5
          </span>
        </div>
```

Follow BEM principles